styles.css

```diff
@@ -19,7 +19,32 @@ body {
 
 .logo {
   font-family: 'PT Serif', serif;
+  font-size: 24px;
+  line-height: 31px;
+  position: relative;
   padding: 40px 0;
+  padding-left: 44px;
+}
+
+.logo__circle {
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+}
+
+.logo__circle_color_dark-blue {
+  background: #0056EC;
+}
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
```

Comment:
Codacy found an issue: [Expected "#C8DBFB" to be "#c8dbfb" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/newpilot/pullRequest?prid=6260226)